src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-feed for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-feed/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-feed/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Feed\Reader\Extension\PodcastIndex;
+
+use Laminas\Feed\Reader\Extension;
+use stdClass;
+
+class Feed extends Extension\AbstractFeed
+{
+    /**
+     * Is the podcast locked (not available for indexing)?
+     *
+     * @return bool
+     */
+    public function isLocked()
```

Comment:
The DocBlock for the parameters, return type and so on can be omitted if the same informations is already included in the method signature.

```suggestion
    /**
     * Is the podcast locked (not available for indexing)?
    public function isLocked(): bool
```